client/my-sites/plugins/main.jsx

```diff
@@ -417,19 +434,22 @@ export default React.createClass( {
 			return;
 		}
 
+		const inBulkEditMode = this.state.bulkManagement.indexOf( isWpCom ? 'wpcom' : 'jetpack' ) !== -1 ? true : false;
```

Comment:
see above note on `includes`

N.B. This is coming in ES7 as native and is already in stage 4 of the ECMA process, so once that happens it will be easier

```js
const inBulkEditMode = ! bulkManagement.includes( isWpCom ? 'wpcom' : 'jetpack' );
```

I think this code will be easier to find if we use `includes()` from `lodash` now, though using that in and of itself is a win in my opinion because of how it reads